hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/UniqueTokenManagementSpecs.java

```diff
@@ -772,7 +772,8 @@ private HapiSpec uniqueWipeFailsWhenInvokedOnFungibleToken() { // invokes unique
                                 .hasKnownStatus(INVALID_WIPING_AMOUNT)
                                 .via("wipeTx"),
                         wipeTokenAccount(A_TOKEN, ACCOUNT, List.of())
-                                .hasKnownStatus(OK)
+                                .hasPrecheck(OK)
+                                .hasKnownStatus(SUCCESS)
```

Comment:
`hasKnownStatus(OK)` makes no sense because it is not a final status, merely a precheck status. Mono service, oddly enough, passes either way, so we'll go with the test that makes sense.

Reply:
We don't need `.hasPrecheck(OK)` , since we are asserting `.hasKnownStatus(SUCCESS)`

Reply:
Done